libc/vga/writev-vga.c

```diff
@@ -58,9 +52,194 @@ ssize_t sys_writev_vga(struct Fd *fd, const struct iovec *iov, int iovlen) {
   return wrote;
 }
 
+static void _vga_init_test(void *vid_buf, unsigned char vid_type,
```

Comment:
This looks like code we'll probably want to TODO move into examples/ soon. This isn't needed for production code, correct?

Reply:
@jart : yes, it is not really needed.  Thank you!